client/app/bundles/HelloWorld/components/Filters.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class Filters extends React.Component {
+    static propTypes = {
+        cuisines: PropTypes.array.isRequired,
+        updateCuisineFilter: PropTypes.func.isRequired,
+        updateRatingFilter: PropTypes.func.isRequired
+    };
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            cuisines: this.props.cuisines
```

Comment:
You can also use: `['All', ...cuisines: this.props.cuisines] ` and then you'll be able to set the initialize state outside the ctor (and remove the ctor)